astroquery/mast/tests/test_mast.py

```diff
@@ -372,6 +372,16 @@ def test_observations_download_products(patch_post, tmpdir):
     assert isinstance(result, Table)
 
 
+def test_observations_download_file(patch_post, tmpdir):
+    # pull a single data product
+    products = mast.Observations.get_product_list('2003738726')
+    uri = products['dataURI'][0]
+
+    # download it
+    result = mast.Observations.download_file(uri)
+    assert result == ('COMPLETE', None, None)
+
+
```

Comment:
This test fails I think because the downloaded file can't be found, not sure if the patching is working correctly. Let me know if you need help sorting this one out.

Reply:
Yeah I couldn't run these tests locally so it was hard to test this.  I tried to follow the examples of the related tests for `test_download_products`, etc, but maybe I'm missing something.

Reply:
I'll look at it, writing patched tests is really not intuitive.

Reply:
I think I figured it out.  Tests locally pass at least and pushing now to test on Travis.

Reply:
Looks good.